datalad/cmdline/main.py

```diff
@@ -67,9 +67,15 @@ class ArgumentParserDisableAbbrev(argparse.ArgumentParser):
     # Don't accept abbreviations for long options. With py3.5 and above, we
     # could just use allow_abbrev=False.
     #
-    # See https://bugs.python.org/issue14910#msg204678
+    # Modified from the solution posted at
+    # https://bugs.python.org/issue14910#msg204678
     def _get_option_tuples(self, option_string):
-        return []
+        chars = self.prefix_chars
+        if option_string[0] in chars and option_string[1] in chars:
```

Comment:
should it get `len(option_string)>1` as the first conditional for paranoid me?

Reply:
As it is now mirrors the exact and first conditional in `ArgumentParser._get_option_tuples`, so I'd bet we're safe.

Reply:
well, we are sticking our fingers into some privates of the argparse... they might change their views/implementation there without letting us know -- well, sure thing we could fix up for it when time comes

Reply:
> well, sure thing we could fix up for it when time comes

Hopefully before then datalad has dropped py2 support and can use `allow_abbrev=False` :]